about.css

```diff
@@ -0,0 +1,434 @@
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  background-color: white;
+}
+
+#header a {
+  color: black;
+  text-decoration: none;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
+
+.session-container {
+  display: none;
+}
+
+.secondary-nav {
+  display: none;
+}
+
+#menu {
+  position: absolute;
+  z-index: 1;
+}
+
+.close-menu-icon {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+}
+
+#menu-container {
+  position: absolute;
+  font-weight: bold;
+  top: 4em !important;
```

Comment:
- Hey, as suggested by the previous reviewer [here](https://github.com/Maedea9/capstone-m1/pull/3/files/1332d8d364cd7085207356e5f74b9a48814a086a#r1142819953), let's a better way of applying styles instead of using the `!important` keyword. please, fix this. they are many that's why he only highlighted one. You should fix this on all occurece in styles except if you're using a CSS framework.

Reply:
It is fixed thanks!

Reply:
- [ ] It is resolved